src/app/router.tsx

```diff
@@ -31,6 +32,10 @@ const router = createHashRouter([
     path: '/auth/get-started',
     element: <GetStarted />,
   },
+  {
```

Comment:
as the figma shows the setnewpassword more like a modal than a new page

Reply:
@LH-Song New commit with Popup window ready to be reviewed.